spec/footer_spec.rb

```diff
@@ -1,4 +1,5 @@
 require_relative '../app/pages/page methods/mega-menu_met'
+require_relative '../app/assertions/footer_assert'
 
```

Comment:
1. Add the required_relative for _LoginMethods_ class, in order to compile code and class be visible on this class.